tuner/tuner/candidate_gen.py

```diff
@@ -187,20 +187,232 @@ def __str__(self) -> str:
 
 
 @dataclass
-class Configuration:
+class BaseConfiguration:
     subgroup_size: int
     workgroup_size: list[int]
-    intrinsic: MfmaIntrinsic
     tile_sizes: list[int]
     subgroup_m_count: int
     subgroup_n_count: int
```

Comment:
I think that all the base configuration needs is tile sizes? For example on CPU this would be the only thing to tune. `subgroup_m_count` and `subgroup_n_count` are specific to `LLVMGPUVectorDistribute`